src/components/app/Home.js

```diff
@@ -491,41 +519,63 @@ class HomeImpl extends React.PureComponent<HomeProps, HomeState> {
           {specialMessage ? (
             <div className="homeSpecialMessage">{specialMessage}</div>
           ) : null}
-          <p>
-            Capture a performance profile. Analyze it. Share it. Make the web
-            faster.
-          </p>
+          <Localized id="Home--special-message">
+            <p>
+              Capture a performance profile. Analyze it. Share it. Make the web
+              faster.
+            </p>
+          </Localized>
           <TransitionGroup className="homeInstructionsTransitionGroup">
             {this._renderInstructions()}
           </TransitionGroup>
           <section className="homeAdditionalContent">
             {/* Grid container: homeAdditionalContent */}
             <h2 className="homeAdditionalContentTitle protocol-display-xs">
               {/* Title: full width */}
-              Load existing profiles
+              <Localized id="Home--additional-content-title">
+                Load existing profiles
+              </Localized>
             </h2>
             <section className="homeActions">
               {/* Actions: left column */}
-              <p>
-                You can <strong>drag and drop</strong> a profile file here to
-                load it, or:
-              </p>
+              <Localized
+                id="Home--additional-content-content"
+                elems={{ strong: <strong /> }}
+              >
+                <p>
+                  You can <strong>drag and drop</strong> a profile file here to
+                  load it, or:
+                </p>
+              </Localized>
               <ActionButtons
                 // $FlowFixMe Error introduced by upgrading to v0.96.0. See issue #1936.
                 retrieveProfileFromFile={this.props.retrieveProfileFromFile}
                 triggerLoadingFromUrl={this.props.triggerLoadingFromUrl}
               />
-              <p>
-                You can also compare recordings.{' '}
-                <InnerNavigationLink dataSource="compare">
-                  Open the comparing interface.
-                </InnerNavigationLink>
-              </p>
+
+              <Localized
+                id="Home--compare-recordings-info"
+                elems={{
+                  InnerNavigationLink: (
```

Comment:
Suggestion: use `<a>` instead of `<InnerNavigationLink>` for the localized string, so that it's clear for the localizers that we're having a link. The fact that we implement this with `InnerNavigationLink` is a detail they don't need to know.

Reply:
Okay but what about the prop `dataSource` passed in the `innerNavigationLink`?